x-pack/plugins/ml/public/application/services/ml_api_service/trained_models.ts

```diff
@@ -196,6 +196,14 @@ export function trainedModelsApiProvider(httpService: HttpService) {
         query: { type, node, showClosedJobs },
       });
     },
+
+    putConfig(modelId: string, config: object) {
```

Comment:
nit, `putTrainedModelConfig` or just `putTrainedModel` is more consistent with the other function names

Reply:
renamed in https://github.com/elastic/kibana/pull/155867/commits/e407bcba2f575a23afb76592c7321878e9d092f7